.github/actions/do_build_pr/run_ubuntu_gcc_x86_64_riscv_fp16_cl3_0_unitcl_vecz/action.yml

```
@@ -0,0 +1,31 @@
+name: build_pr_ubuntu_gcc_x86_64_riscv_fp16_cl3_0_unitcl_vecz
+description: Build pr ubuntu_gcc_x86_64_riscv_fp16_cl3_0_unitcl_vecz
+
+inputs:
+  cache_seed:
+    type: boolean
+    default: false
+
+runs:
+  using: "composite"
+  steps:
+      - name: remove any old dirs
+        shell: bash  
+        run:
+          rm -rf build
+
+      - name: build ock
+        uses: ./.github/actions/do_build_ock
+        with:
+          build_targets: ${{ inputs.cache_seed == 'true' && 'clc UnitCL' || 'check-ock-UnitCL-group-vecz' }}
```

As `inputs.cache_seed` is a boolean rather than a string,
```suggestion
          build_targets: ${{ inputs.cache_seed && 'clc UnitCL' || 'check-ock-UnitCL-group-vecz' }}
```
should be sufficient? Likewise in many other places. But I'm not entirely sure of what data types exist in GitHub actions, and if this suggestion does not work, then what you have already is fine.

---

I thought that too, but it doesn't work - it even goes wrong without the quotes.